packages/features/insights/server/trpc-router.ts

```diff
@@ -1651,12 +1645,16 @@ export const insightsRouter = router({
       })
     )
     .query(async ({ ctx, input }) => {
-      return await RoutingEventsInsights.getRoutingFormHeaders({
+      const fields = await RoutingEventsInsights.getRoutingFormHeaders({
```

Comment:
Naming wise this causes me some confusion:

```suggestion
      const headers = await RoutingEventsInsights.getRoutingFormHeaders({
```